corehq/motech/migrations/0014_fix_whitelist_bug_repeaters.py

```diff
@@ -0,0 +1,18 @@
+from django.db import migrations
+
+from corehq.motech.migration_utils import repair_repeaters_with_whitelist_bug
+
+
+def _fix_broken_repeaters(apps, schema_editor):
```

Comment:
Minor suggestion, but this could be skipped on fresh install.
```suggestion
from corehq.util.django_migrations import skip_on_fresh_install


@skip_on_fresh_install
def _fix_broken_repeaters(apps, schema_editor):```